src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -87,10 +89,15 @@ public bool CheckBoxesEnabled
         public PackageItemListViewModel SelectedPackageItem => _list.SelectedItem as PackageItemListViewModel;
 
         // Load items using the specified loader
-        internal void LoadItems(IItemLoader<PackageItemListViewModel> loader, string loadingMessage, INuGetUILogger logger)
+        internal void LoadItems(
+            PackageItemLoader loader, 
```

Comment:
Agree, but i didn't think of a case where there will be multiple implementations for this IItemLoader, besides there is no such contract defined between IItemLoader and InfiniteScrollList. So I went ahead with PackageItemLoader class itself. But giving it further thoughts, I can define a new interface which will make it more extensible.